extensions/positron-proxy/package.json

```diff
@@ -22,14 +22,14 @@
     "menus": {
       "explorer/context": [
         {
-          "when": "resourceLangId == html",
+          "when": "resourceLangId == html && resourceScheme == file",
```

Comment:
since the tab is technically not "looking at a file", this allows source html to keep the eye icon to open in viewer, but it should not show up at all for viewer->editor tabs!

Reply:
Works great!